tools/isobuild/bundler.js

```diff
@@ -184,6 +185,62 @@ exports.ignoreFiles = [
       */
 ];
 
+const METEOR_IGNORE = '.meteorignore'
+
+/*
+    .meteorignore works *like*
+    https://git-scm.com/docs/gitignore
+    what's implemented: 
+      - blank line separator for readability
+      - A line starting with # as a comment, \# or [#] is not comment.
+      - Trailing spaces are ignored.
+      - glob stars ** matches all directory. 
+        for example trailing /** matches eveying inside   */   
+//       a/**/b matches a/b a/x/b, a/x/y/b...  
+/*
+    what's implemented but not in the specification.
+      - ? for match single character
+      - [] to match character group.
+
+    What's not effective: (doesn't work.)
+      - ! negative pattern won't be effective unless the below known issue is resolved.
+
+    known issue:
+      ignoreFiles take regExp list with default logical *or*, 
+      instead of glob pattern list with optional logical *and* for negative pattern.
+        *        # ignore all files
+        !*.html  # but (AND) not ignore *.html, ignore files that matches * AND !*.html
+        !*.css   # (AND) not ignore *.css
+        !*.js    # (AND) not ignore *.js
+*/
+
+if (files.exists(METEOR_IGNORE)) {
```

Comment:
Taking inspiration from how `.gitignore` files work, I think it's important to allow multiple `.meteorignore` files in different directories, each governing the files contained by its own directory tree.

It's also really important that we allow `.meteorignore` files to be added/moved/removed while the development server is running, so it's not enough to check the file once, as you're doing here.

I would strongly encourage you to consider implementing these changes in the [`PackageSource#_findSources`](https://github.com/i4han/meteor/blob/9a0b09b2159a190b3a8241620493e4426ea3b089/tools/isobuild/package-source.js#L984) method, instead of doing it at the top-level here.